README.md

```diff
@@ -3,4 +3,104 @@
 [![codecov](https://codecov.io/gh/velocidi/velocidi-android-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/velocidi/velocidi-android-sdk)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.velocidi/velocidi-android-sdk/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.velocidi/velocidi-android-sdk)
 
-Tools to integrate Android Apps with Velocidi Private CDP
+Open-source SDK to integrate Android Apps with Velocidi Private CDP.
+
+## Installation
+
+Velocidi SDK is published to Maven central as a single module. Please reference the badge above or go to the [release page](https://github.com/velocidi/velocidi-android-sdk/releases) to check our latest version.
+
+To install the SDk in your application you just need to add the following dependency to the `app/build.gradle` file.
+
+```
+dependencies {
+    implementation 'com.velocidi:velocidi-android-sdk:0.0.1'
+}
+```
+
+You also need to add the following permissions to ensure the good functioning of the SDK.
```

Comment:
Because this is only needed for apps targeting SDK <= 22 (Android 5.1 or less/ As of 6.0, Internet permissions is granted by default) I would move this to another section with the proper indication that this is no longer needed.

Reply:
As discussed offline, the only change in recent SDK is that internet permissions aren't prompt to the user but applications still require the permission in the manifest.